sdk/storage/azure-storage-common/inc/azure/storage/common/account_sas_builder.hpp

```diff
@@ -103,7 +103,11 @@ namespace Azure { namespace Storage { namespace Sas {
      * access signature.
      */
     Files = 4,
-
+    /**
+     * @brief Indicates whether Azure Table Storage resources are accessible from the shared
+     * access signature.
+     */
+    Table = 8,
```

Comment:
Add an entry to the Storage Common's README.md Other Changes section:
``- Added support for `azure-data-tables` connection strings.``